Cargo.toml

```
@@ -16,6 +16,13 @@ cidr = "0.2"
 lazy_static = "1.4"
 uuid = "1.5"
 regex = "1.10"
+serde = { version = "1.0", features = ["derive"], optional = true }
+serde_regex = { version = "1.1.0", optional = true }
```

```suggestion
serde_regex = { version = "1.1", optional = true }
```

To allow dependabot to bump dependency versions.